core/src/main/scala/stryker4s/extension/mutationtype/ConditionalExpression.scala

```diff
@@ -0,0 +1,18 @@
+package stryker4s.extension.mutationtype
+
+import scala.meta._
+
+case object If {
+
+  def unapply(term: Term): Option[Term] =
+    term.parent collect {
```

Comment:
Thinking about it a bit more: Why not something like this:

```scala
def unapply(if: Term.If): Option[Term] = Some(if.cond).filterNot(_.is[Lit.Boolean])
```

That way the unapply matches on a Term.If, instead of checking every node if it has an If as a parent. Which is probably not very efficient

Reply:
This does result in mutating the entire if-statement instead of just the condition, because then the `orig` in `case If(orig) => orig ~~> (IfTrue, IfFalse)` will be the whole if-statement. We might be able to work around that but I don't immediately see how...

Reply:
Would it? You make an Option of the condition (`Some(ifStatement.cond)`). Wouldn't that result in the `orig` being the condition? And that then being mutated to true & false

Reply:
Actually you are correct, just tested it. :') Good catch.